source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -2260,6 +2261,19 @@ message TransformationSwapConditionalBranchOperands {
 
 }
 
+message TransformationSwapFunctionVariables {
+	// A transformation that swaps function variables 
+	// set all second constuctor values 
```

Comment:
I don't know what this comment line means.

Reply:
here I've just described Vars in message represents a constructor Args of "TransformationSwapFunctionVariables" class.